app/js/services/resultatService.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-angular.module('ddsApp').service('ResultatService', function($http, droitsDescription) {
+angular.module('ddsApp').service('ResultatService', function($http, $modal, droitsDescription) {
```

Comment:
Ok, I think I got it. Something like [this](http://web2.cloudmanic.com/~skitch/Contacts-20130514-003837.jpg) right ?